test/erlcloud_kms_tests.erl

```diff
@@ -173,6 +174,7 @@ output_test(Fun, {Line, {Description, Response, Result}}) ->
      {Line,
       fun() ->
               meck:expect(erlcloud_httpc, request, output_expect(Response)),
+              erlcloud_kms:configure(string:copies("A", 20), string:copies("a", 40)),
```

Comment:
I believe this triggers setting the AWS config in the process dictionary, so that calls to [default_config](https://github.com/rymir/erlcloud/blob/kms-iam-role/src/erlcloud_aws.erl#L193-L208) work correctly. It seems to be SOP in most of the unit tests I have seen.